Rakefile

```diff
@@ -80,6 +80,9 @@ Rake::TestTask.new("test:unit") do |t|
   ENV["RUBYOPT"].split(/\s/).each do |opt|
     t.ruby_opts << opt
   end
+  if t.ruby_opts.none? { |x| x =~ /^\-W/ }
```

Comment:
Stop showing warnings - some of the code in the tests does stuff you wouldn't want in prod code but I don't feel like fixing it and don't think they are issues. `RUBYOPT` can be set explicitly to avoid this behavior